core/src/main/java/org/apache/accumulo/core/metadata/schema/TabletMetadata.java

```diff
@@ -527,13 +492,13 @@ public static <E extends Entry<Key,Value>> TabletMetadata convertRow(Iterator<E>
               BulkFileColumnFamily.getBulkLoadTid(val));
           break;
         case CurrentLocationColumnFamily.STR_NAME:
-          te.setLocationOnce(val, qual, LocationType.CURRENT, suppressLocationError);
+          te.setLocationOnce(val, LocationType.CURRENT, suppressLocationError);
           break;
         case FutureLocationColumnFamily.STR_NAME:
-          te.setLocationOnce(val, qual, LocationType.FUTURE, suppressLocationError);
+          te.setLocationOnce(val, LocationType.FUTURE, suppressLocationError);
           break;
         case LastLocationColumnFamily.STR_NAME:
-          te.last = Location.last(val, qual);
```

Comment:
What is the benefit of pushing the tservers group into the metadata table for each tablet?

Reply:
It would be needed by the Manager to know whether or not a table's tablets are balanced in the correct group.

Reply:
The manager could get the group information about each tserver w/o it being stored in each tablet.